src/_gettsim_personas/de/grundsicherung_für_erwerbsfähige/__init__.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from pathlib import Path
+
+from _gettsim_personas.persona_objects import Persona
+
+Couple1Child = Persona(
+    path_to_persona_elements=Path(__file__).parent / "_couple_1_child.py",
+    start_date=datetime.date(2005, 1, 1),
+    error_if_not_implemented=(
+        "Personas are available from 2005. Basic income support is not implemented in "
```

Comment:
```suggestion
        "These personas are available from 2005 because basic income support is not implemented in "
```
Sounds too general otherwise if I am not familiar with the codebase.